src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/11.1-1.sql

```diff
@@ -104,7 +104,7 @@ BEGIN
     --
     -- register triggers
     --
-    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition
+    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition JOIN pg_class ON (logicalrelid = oid) WHERE relkind <> 'f'
```

Comment:
we'd also need to backport this to 11.0-X.


You'd kind of need another PR like this: https://github.com/citusdata/citus/pull/6037/files